src/install/haproxy/whatsNext.mdx

```diff
@@ -0,0 +1,1766 @@
+---
+componentType: default
+headingText: Metrics collected by the integration
+---
+
+## Example configurations [#examples]
```

Comment:
I believe this section is what you mean to reference when you direct ppl to update the config file. Maybe we can move these collapsers directly into that step for all the options? A little bit of extra work but it sidesteps having customers overlook the section + not update at all, or misunderstand that they need to.

That said, maybe they don't NEED to. If it's optional, then I'd at least specify when they'd want to use basic config over the one that gets copied into the directory